.github/workflows/flutter-test.yml

```
@@ -25,5 +25,7 @@ jobs:
       - run: echo "::add-path::$GITHUB_WORKSPACE/_flutter/bin"
       - run: flutter pub get
         working-directory: ./client/flutter
+      - run: flutter analyze
+        working-directory: ./client/flutter
       - run: flutter test
```

```suggestion
      - run: flutter test --no-pub
```
(I realize this is not part of analysis, but might as well add it in this PR)